src/App.tsx

```diff
@@ -1,222 +1,215 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Header } from './components/Header/Header';
+import { Footer } from './components/Footer/Footer';
+import { TodoList } from './components/TodoList/TodoList';
+import { Errors } from './types/Errors';
+import { Filter } from './types/Filter';
+import { Todo } from './types/Todo';
+import {
+  getTodos,
+  addTodo,
+  removeTodo,
+  updateTodo,
+} from './api/todos';
+// eslint-disable-next-line max-len
+import { ErrorNotification } from './components/ErrorNotification/ErrorNotification';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
 
+  const [isError, setIsError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState<Errors>(Errors.NONE);
+  const [filterBy, setFilterBy] = useState<Filter>(Filter.All);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoTemp, setTodoTemp] = useState<Todo | null>(null);
+  const [todosFiltered, setTodosFiltered] = useState<Todo[]>([]);
+  const [isAdding, setIsAdding] = useState(false);
+  const [activeTodoID, setActiveTodoID] = useState<number[]>([]);
+
+  const closeError = () => setIsError(false);
+
+  const showError = (message: Errors) => {
+    setIsError(true);
+    setErrorMessage(message);
+    setTimeout(closeError, 3000);
+  };
+
+  const loadTodos = useCallback(async () => {
+    try {
+      if (user) {
+        await getTodos(user.id)
+          .then(data => setTodos(data));
```

Comment:
let's not use `then` method inside `try/catch` statement. just set a value using a separate variable
`const todosFromServer = await getTodos(user.id);
setTodos(todosFromServer);`